RecoTracker/TkTrackingRegions/plugins/AreaSeededTrackingRegionsProducer.h

```diff
@@ -0,0 +1,118 @@
+#ifndef RecoTracker_TkTrackingRegions_AreaSeededTrackingRegionsProducer_h
```

Comment:
[minor/aside remark] "Generator" or "Maker" in the name may be a good disambiguation from mistaking this class name for an EDProducer.

Reply:
In general I agree, but a change in the naming convention (originating from 2007...) should be considered separately from this PR. On the other hand the relatively recent change of producing `TrackingRegions` to `Event` increased the possibilities for confusion.